lib/firebase-adapter.js

```diff
@@ -2,7 +2,7 @@ var firepad = firepad || { };
 
 firepad.FirebaseAdapter = (function (global) {
 
-  if (window.firebase === undefined && typeof require === 'function' && typeof Firebase !== 'function') {
+  if (typeof require === 'function' && typeof Firebase !== 'function') {
```

Comment:
Added this back in. Also, somewhat unrelated, but it looked like the tests were just failing due to Jasmine timeouts. I have been playing around with modifying that timeout so test will pass.